conanfile.py

```diff
@@ -147,3 +147,6 @@ def package_info(self):
             self.cpp_info.libs.extend(["winhttp", "httpapi", "bcrypt"])
         if not self.options.shared:
             self.cpp_info.defines.append("_NO_ASYNCRTIMP")
+        if self.settings.os == "Macos":
+            self.cpp_info.exelinkflags.append("-framework CoreFoundation")
+            self.cpp_info.exelinkflags.append("-framework Security")
```

Comment:
they also required to be added to `self.cpp_info.sharedlinkflags`

Reply:
Done